### PR TITLE
PanelTabOptions WIP

### PR DESCRIPTION
**Pull request by JoshRosenstein**
Issue: 
no options to customize panel tabs. similar to `showPanel`, allow user to config Panel tabs globally or per story.
## What I did
- added two options to layout api, `hiddenTabs` & `tabsOrder` (names up for discussion), in order to customize panel tabs.
- Add sorter method within panel component

***Note, having trouble building (on windows), will rely on CI for now , therefore this PR may be temporary.
 
## How to test

- Is this testable with Jest or Chromatic screenshots?
- Does this need a new example in the kitchen sink apps?
maybe?
- Does this need an update to the documentation?
yep

If your answer is yes to any of these, please make sure to include it in your PR.

### CODE REVIEW

File: lib/ui/src/components/panel/panel.js [hunk 1]
```diff
@@ -10,6 +10,20 @@ const DesktopOnlyIconButton = styled(IconButton)({
   },
 });
 
+const tabSorter = (panels, panelOrder) => {
```
Comment (tmeasday) [inline lib/ui/src/components/panel/panel.js]: Could we pull this out into a separate lib file and and a couple of simple tests? (Perhaps coupled with the hidden logic too?)

**Review by tmeasday — COMMENTED** [review]
This is great! I think we'll need to solve #5857 before it can really be used however

File: lib/ui/src/components/panel/panel.js (hunk 1 above)
Comment (JoshRosenstein) [reply lib/ui/src/components/panel/panel.js]: @tmeasday  Sure thing.

**Comment by ndelangen** [comment]
I think this feature is the same that was recently added by @atanasster.

**Pull request closed without merging**